Documentation/ceph-object.md

```diff
@@ -105,6 +105,46 @@ To directly retrieve the secrets:
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep AccessKey | awk '{print $2}' | base64 --decode
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep SecretKey | awk '{print $2}' | base64 --decode
 ```
+## Create an Object Bucket Claim
+
+Creating an Object Bucket Claim (OBC) causes the Rook-Ceph bucket provisioner to create a new bucket or to grant access to an existing bucket. When the OBC does not references an existing storage class set up by an administrator, a new bucket is created with the specified retention policy. See the [Object Bucket Claim Documentation](ceph-object-bucket-claim.md) for more detail on the `CephObjectBucketClaims`.
+
+```yaml
```

Comment:
What is the reason to repeat the yamls here? Additionally, this order of the yamls here is different from the other doc.

Reply:
I would delete L113 - L147 unless there is info unique to this doc.